prow/plugins.yaml

```diff
@@ -85,8 +85,9 @@ plugins:
   - trigger
 
   kubernetes/community:
-  - trigger
+  - approve
```

Comment:
Note that since you didn't specify an 'approve' config for k/community the defaults will be used:
- no associated issue will be required
- no implicit self-approval from PR authors

Are these defaults okay?

Reply:
Yes, these are the defaults I was looking for. Thanks!